### ISSUE TITLE
Restart count inconsistent.

### ISSUE THREAD

**Issue by ericrini** [Unitech/pm2#1972]
Something I observed. Possibly a bug.

```
pm2 status
```

The "restart" column for process 1 shows 0.

```
pm2 restart 1
```

EXPECTED: The "restart" column for process 1 shows 1.
ACTUAL: The "restart" column for process 1 shows 1.

```
pm2 stop 1
pm2 restart 1
```

EXPECTED: The "restart" column for process 1 shows 2.
ACTUAL: The "restart" column for process 1 shows 1.

**Comment by sholladay** [comment]
The output makes perfect sense to me. A restart is a stop and then a start. You even ran `restart` twice.

**Comment by ericrini** [comment]
Right, the complaint is that the restart column shows 1 restart after stopping and starting the process 2 times in total. It only increments if the process was running when restarted. Not any time it was started from a stopped state.

**Comment by sholladay** [comment]
I must've read that backwards last night. DIsregard. :)
